plugins/dashboard/frontend/src/app/components/ConfirmedMilestoneChart.tsx

```diff
@@ -130,10 +132,21 @@ export default class ConfirmedMilestoneChart extends React.Component<Props, any>
                 <Card.Body>
                     <Card.Title>Confirmed Milestones</Card.Title>
                     <If condition={!!this.props.nodeStore.last_confirmed_ms_metric.ctps}>
-                        <small>
-                            CTPS: {(this.props.nodeStore.last_confirmed_ms_metric.ctps).toFixed(2)}.
-                            Confirmation: {(this.props.nodeStore.last_confirmed_ms_metric.conf_rate).toFixed(2)}%
-                        </small>
+                        <Row>
+                            <Col>
+                                <small>
+                                    CTPS: {(this.props.nodeStore.last_confirmed_ms_metric.ctps).toFixed(2)}.
+                                    Confirmation: {(this.props.nodeStore.last_confirmed_ms_metric.conf_rate).toFixed(2)}%
+                                </small>
+                            </Col>
+                            <Col>
+                                <small>
+                                    Avg. TPS: {(this.props.nodeStore.confirmedMilestonesSeries.datasets[0].data.reduce((a, b) => a + b) / this.props.nodeStore.confirmedMilestonesSeries.datasets[0].data.length).toFixed(2)}.
```

Comment:
@Thoralf-M why use the series and not the `collected_confirmed_ms_metrics`?

Reply:
Didn't know that it exists